mapper/mapper.go

```diff
@@ -71,8 +71,8 @@ func ReleaseCalendar(rawReleaseCalendar release_calendar.ReleaseCalendar) *model
 		}
 	}
 	rc := model.ReleaseCalendar{
-		Releases:         getLatestReleases(releaseResults),
-		NumberOfReleases: strconv.Itoa(numReleasesScheduled),
+		Releases:                         getLatestReleases(releaseResults),
```

Comment:
Won't hold up approval but maybe we could set a variable e.g. `latestReleases` to store what `getLatestReleases` returns - and store that in the struct instead; then for the calculation we could just have `numReleasesScheduled - len(latestReleases)` instead of having the function call again